relationships/synthesis/APM-APPLICATION-to-INFRA-MSSQLINSTANCE.yml

```diff
@@ -25,10 +26,11 @@ relationships:
           identifier:
             fragments:
               - value: "ms-instance:"
-              - attribute: metricName__4 # hostname
-              - value: ":instance="
-              # This rule assumes that the instance name is equal to the db hostname in lower case.
-              - attribute: metricName__4 # hostname in lowerCase
+              - capture:
```

Comment:
The identifier is being changed from `ms-instance:abc123:instance:abc123` to just `ms-instance:abc123`.
Just to double check, is that the expected behaviour?

Reply:
hey @javimb  - I admit I don't understand the naming convention of MS SQL instances (and many other entity names), but that wasn't my intention to change the identifier because I don't intend on updating the Entity Synthesis model for MS SQL instances.

I think I've corrected the PR... or at least taken a step in the right direction by having two capture groups for the name of the MS instance so we can match the syntax `ms-instance:ABC123:instance:abc123` where ABC123 may or may not be in uppercase, but the latter abc123 is always lowercased.

Please let me know if this looks correct and I will have someone from core integrations team give it another look over.